mdx-web/src/main/java/com/mx/path/model/mdx/web/controller/AccountNumbersController.java

```diff
@@ -19,4 +19,11 @@ public final ResponseEntity<AccountNumbers> get(@PathVariable("id") String accou
     AccessorResponse<AccountNumbers> response = gateway().accounts().accountNumbers().get(accountId);
     return new ResponseEntity<>(response.getResult().wrapped(), createMultiMapForResponse(response.getHeaders()), HttpStatus.OK);
   }
+
+  @RequestMapping(value = "/users/{user_id}/accounts/{id}/account_number", method = RequestMethod.GET, produces = BaseController.MDX_MEDIA)
+  public final ResponseEntity<AccountNumbers> getMdx(@PathVariable("user_id") String userId, @PathVariable("id") String accountId) throws Exception {
```

Comment:
Can we change this method name to `getAccountNumbers`? The controller methods are usually on the verbose side and include the name of the model being returned.

I think the OnDemand method name should also be changed to `getOnDemandAccountNumbers` to be more explicit.

Reply:
Yes, I suggested that name above in another thread as one of the ones I was already considering. As for on demand, I don't want to touch that as it would potentially require multiple version updates to phase it out. Unless someone can confirm it is definitely not being used (in which case not sure why it's there in the first place). I know you asked about that and didn't see it in GitLab, but I definitely would prefer not to break other clients, even if its a small chance.

Reply:
Changing a method name in the controller should be safe. Changing a method name in the base accessor can be problematic because those are implemented in accessor libraries.

@mattnichols Can you think of any potential issues with changing a controller method name?

Reply:
Right. That makes sense, it seems to me like the determining factor for the controllers is the request mapping. The name isn't as big a deal. Agreed that with the accessor library that's a different story, but since both methods here are gonna use the simple `get()`, I think we are good there. 

I'll make that change to the ondemand controller method unless Matt thinks otherwise, thanks!